### PR TITLE
[Banner] Added BannerCfg class to hostcfgd to handle Banner messages

### PR DESCRIPTION
**Pull request by SviatoslavBoichuk**
#### Why I did it
Added Banner feature related hostcfgd service according to HLD: https://github.com/sonic-net/SONiC/pull/1361

This PR depends on the following PR: https://github.com/sonic-net/sonic-swss-common/pull/826

#### How I did it
Added BannerCfg service to hostcfgd. It handles Config DB changes and restart "banner-config" systemd service which will update banners.

#### How to verify it
Manual test

### CODE REVIEW

**Comment by liat-grozovik** [comment]
@SviatoslavBoichuk  please handle conflict. Please also add description of the code changes and motivation

**Comment by SviatoslavBoichuk** [comment]
> @SviatoslavBoichuk please handle conflict. Please also add description of the code changes and motivation

Fixed conflicts, updated PR description

File: scripts/hostcfgd [hunk 1]
```diff
@@ -1500,6 +1500,85 @@ class FipsCfg(object):
...
+        Apply banner message handler.
+
+        Args:
+            cache: Cache to compare/save data.
+            db: DB instance.
+            table: DB table that was changed.
+            key: DB table's key that was triggered change.
+            data: Read table data.
+        """
+        # Handling state, login/logout and MOTD messages. Data should be a dict
```
Comment (qiluo-msft) [inline scripts/hostcfgd:1555]: >state [](http://example.com/codeflow?start=19&length=5)

Where is the code to handle `state`? If the feature is disabled, we expected that no extra daemon started, minimal CPU wasted in this script.

Comment (SviatoslavBoichuk) [reply scripts/hostcfgd:1555]: The "state" is used by "banner-config.service". Hostcfgd responsible only to track DB changes for BANNER table and restart banner service once user modified banner messages or feature state. The banner service (please see PR: https://github.com/sonic-net/sonic-buildimage/pull/16957/) will apply new Banners to Linux only if feature state == enabled.

Comment (qiluo-msft) [reply scripts/hostcfgd:1555]: How to disable the feature? In normal deployment, if people do not want use this feature, would like to disable it completely, minimize teh CPU wasted relating to this feature.

Comment (SviatoslavBoichuk) [reply scripts/hostcfgd:1555]: To disable the feature - you can use CLI command: config banner state disabled.

Comment (qiluo-msft) [reply scripts/hostcfgd:1555]: If the feature is disabled, we expected that no extra daemon started, minimal CPU wasted in this script. In this case, is the service "banner-config" still running?

**Comment by fastiuk** [comment]
The test is failing because https://github.com/sonic-net/sonic-swss-common/pull/826 should be merged first

File: scripts/hostcfgd (hunk 1 above)
Comment (fastiuk) [reply scripts/hostcfgd:1555]: @qiluo-msft is that thread resolved?

**Comment by fastiuk** [comment]
Still failing because of https://github.com/sonic-net/sonic-swss-common/pull/826

**Comment by fastiuk** [comment]
@qiluo-msft , added a new unit test. Didn't touch a code. Please re-review/approve

**Comment by fastiuk** [comment]
@qiluo-msft , gentle reminder :)

**Comment by fastiuk** [comment]
@qiluo-msft , gentle reminder :)
Please approve if you have time

**Comment by jhli-cisco** [comment]
Hi @SviatoslavBoichuk we are interested in the banner feature. can it be merged anytime soon?

**Comment by qiluo-msft** [comment]
Please resolve the conflict

**Review by qiluo-msft — APPROVED** [review]

**Comment by fastiuk** [comment]
@qiluo-msft rebased. Please merge

**Comment by fastiuk** [comment]
@qiluo-msft could you please merge it? It was approved by you, I rebased and it passed the CI here

**Comment by fastiuk** [comment]
@qiluo-msft gentle reminder to merge

**Pull request merged**